source/docs/software/advanced-controls/state-space/state-space-observers.rst

```diff
@@ -0,0 +1,120 @@
+State Observers and Kalman Filters
+==================================
+
+State observers combine information about a system's behavior and external measurements to estimate the true :term:`state` of the system. A common observer used for linear systems is the Kalman Filter. Kalman filters are advantageous over other :ref:`filters <docs/software/advanced-controls/filters/index:Filters>` as they fuse measurements from one or more sensors with a state-space model of the system to optimally estimate a system's state.
+
+This image shows flywheel velocity measurements over time, run through a variety of different filters. Note that a well-tuned Kalman filter shows no measurement lag during flywheel spinup while still rejecting noisy data and reacting quickly to disturbances as balls pass through it. More on filters can be found in the :ref:`filters section <docs/software/advanced-controls/filters/index:Filters>`\.
+
+.. image:: images/filter_comparison.png
```

Comment:
We should dig up the data for this plot and recreate it in matplotlib so it's consistent with our other plots.

Reply:
The updated version is in progress [here](https://github.com/mcm001/frc-docs/pull/2).